### ISSUE TITLE
Add deprecated warning to `az acr helm` docs

### ISSUE THREAD

**Issue by joshwright10** [Azure/azure-cli#25316]
When running the `az acr helm` commands, the following warning is shown:

> This command is implicitly deprecated because command group 'acr helm' is deprecated and will be removed in a future release. Use 'helm v3' instead.

This warning should also be added to the documentation page to make it clear before starting to experiment with this set of commands.

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 2bed94c3-b7a7-40d4-0102-8c9b1f1145ca
* Version Independent ID: 2d4ac8bd-dc7a-236c-56db-7f0e1ce0fd05
* Content: [az acr helm](https://learn.microsoft.com/en-us/cli/azure/acr/helm?view=azure-cli-latest)
* Content Source: [latest/docs-ref-autogen/acr/helm.yml](https://github.com/MicrosoftDocs/azure-docs-cli/blob/main/latest/docs-ref-autogen/acr/helm.yml)
* Service: **container-registry**
* GitHub Login: @rloutlaw
* Microsoft Alias: **routlaw**

**Comment by yonzhan** [comment]
route to CXP team

**Comment by PramodValavala-MSFT** [comment]
@joshwright10 Thanks for the update! We will review and update accordingly.

**Comment by jiasli** [comment]
This feature request on documentation is tracked by https://github.com/MicrosoftDocs/azure-docs-cli/issues/2720.

**Comment by toddysm** [comment]
@JXavierMSFT Is this completed? Can we close this issue?

**Comment by JXavierMSFT** [comment]
Yes, we can close this issue. We've added a warning to the Helm docs about the upcoming deprecation of Helm repositories.

**Comment by johnsonshi** [comment]
The helm deprecation warnings have been added to the CLI and the Portal and MS Learn docs: https://learn.microsoft.com/en-us/azure/container-registry/container-registry-helm-repos
